scripts/cleanup-dev-resources.sh

```
@@ -0,0 +1,102 @@
+        docker system prune -f
+        
+        echo "✅ Docker 정리 완료"
+EOF
+fi
+
+echo ""
+echo "✅ 개발 환경 리소스 정리가 완료되었습니다!"
+echo "💰 예상 비용 절약: 월 $10-15"
+echo ""
+echo "📊 정리 요약:"
+echo "  - EC2 인스턴스: $(echo $INSTANCE_IDS | wc -w)개 종료"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)